test/parallel_api/iterator/input_data_sweep.h

```diff
@@ -24,6 +24,29 @@
 #include "support/utils_invoke.h"
 
 #if TEST_DPCPP_BACKEND_PRESENT
+
+template <typename Size, int __reverses>
+struct get_expected_op
+{
+    Size n;
+
+    template <typename T>
+    auto operator()(T exp) const
+    {
+        (void)n; // avoid unused-lambda-capture, bug https://bugs.llvm.org/show_bug.cgi?id=35450
```

Comment:
We can probably remove this line. It looks to be related to lambdas.

Reply:
Fixed